examples/tree/plot_cost_complexity_pruning.py

```diff
@@ -0,0 +1,67 @@
+r"""
+========================================================
+Post pruning decision trees with cost complexity pruning
+========================================================
+
+In this example, decision tree classifiers are trained with a post pruning
+technique called minimal cost complexity pruning. This technique is
+parameterized by the complexity parameter, :math:`\alpha`. Greater values of
+:math:`\alpha` will prune more of the tree, thus creating a smaller trees.
+"""
+
+###############################################################################
+# Train decision tree classifiers
+# -------------------------------
+# Train 40 decision tree classifiers with :math:`\alpha` from 0.00 to
+# 0.40.
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.datasets import load_breast_cancer
+from sklearn.tree import DecisionTreeClassifier
+
+X, y = load_breast_cancer(return_X_y=True)
+X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+
+alphas = np.linspace(0, 0.04, 40)
+clfs = []
+for alpha in alphas:
+    clf = DecisionTreeClassifier(random_state=0, alpha=alpha)
+    clf.fit(X_train, y_train)
+    clfs.append(clf)
+
+###############################################################################
+# Plot training and test scores vs alpha
+# --------------------------------------
+# Calcuate and plot the the training scores and test accuracy scores
```

Comment:
Just a few typos (I'll document myself on tree pruning and  will try to provide mode in-depth review later)

- Calcuate
- the the
- above "a smaller trees"

also I think you should avoid the ``` `:math:` ``` notation in comments

Reply:
The `:math:` notation is currently used in other examples such as: https://github.com/scikit-learn/scikit-learn/blob/master/examples/svm/plot_svm_scale_c.py. Are we discouraging the usage of `:math:` in our examples?

Reply:
It's OK in the docstrings since it will be rendered like regular rst by sphinx, but in the comments it is not necessary.

Reply:
These comments are rendered into html: https://42001-843222-gh.circle-artifacts.com/0/doc/auto_examples/tree/plot_cost_complexity_pruning.html

Reply:
Ooh ok I didn't know it worked like that, sorry